src/PlaneWaveBasis.jl

```diff
@@ -302,3 +318,52 @@ end
 function r_to_G(basis::PlaneWaveBasis, kpt::Kpoint, f_real::AbstractArray3) where {T}
     r_to_G!(similar(f_real, length(kpt.mapping)), basis, kpt, copy(f_real))
 end
+
+#
+# Functions to handle BZ symmetry
+#
+
+"""
+Converts an eigenfunction at point `kpoint_src` to one at
+`kpoint_dst`, the two being related by symmetry operation S.
+"""
+function transfer_ψ(basis::PlaneWaveBasis{T}, kpoint_src::Kpoint, kpoint_dst::Kpoint, ψ::AbstractVector, S, τ) where {T}
```

Comment:
I'm not a fan of the name. It's a bit too generic. From the name it could be related to the things in `interpolation.jl` (which it is in some sense, but not in the sense that it is an interpolation, but exact). Also I think the order of the arguments should be more similar to the way we do in in `interpolation.jl`. I would suggest `transfer_kpoint` or `transfer_to_kpoint` with the argument order `(basis, ψ,  kpoint_in, kpoint_out, S, τ)`

Reply:
completely agree, I didn't like the name either. Will go with transfer_to_kpoint

Reply:
we can move it to interpolations also